src/hypermedia-constants.js

```diff
@@ -553,7 +554,11 @@ export const Classes = {
 	meetings: {
 		bookable: 'bookable'
 	},
-	relativeUri: 'relative-uri'
+	relativeUri: 'relative-uri',
+	ipsis: {
```

Comment:
I'm seeing that we currently have 3 different formats for actions, rels, and classes:

- `Actions.ipsis.sisCourseMerge...`
- `rels.IPSIS.SISCourseMerge...`
- `Classes.ipsis...`

Are we having these different formats on purpose or do we want to try to maintain more consistency?

Reply:
As someone looking at the code for the first time, I wasn't sure why ipsis was capital in one place and not in another.
Not having `sisCourseMerge` inside one and having it in others adds to the confusion.

Reply:
Good point! I will do this in a separate PR as it would mostly clutter this PR. I think we'll move to proper camel case for all of them since that's the typical JS way.